HealthcareSystem/IntegrationAdapters/MicroserviceComunicator/ActionBenefit/ActionBenefitService.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Backend.Model;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IntegrationAdapters.MicroserviceComunicator
+{
+    public class ActionBenefitService : IActionBenefitService
+    {
+        private HttpClient _httpClient;
+
+        public ActionBenefitService(IHttpClientFactory httpClientFactory)
+        {
+            _httpClient = httpClientFactory.CreateClient();
+            _httpClient.BaseAddress = new System.Uri("http://localhost:5002");
```

Comment:
This needs to be configurable, either check how we pulled out all the Service urls into config in PatientWebApp or let me know if we need to handle the configuration.